crates/turborepo/build.rs

```diff
@@ -90,21 +26,38 @@ fn build_debug_libturbo() -> PathBuf {
         for ext in ["pdb", "exe", "d", "lib"].iter() {
             std::fs::remove_file(output_deps.join(format!("turbo.{ext}"))).unwrap_or(());
         }
-
-        cmd.env("CGO_ENABLED", "1")
-            .env("CC", "gcc")
-            .env("CXX", "g++")
-            .arg("turbo.lib");
+        cmd.arg("go-turbo.exe");
     } else {
-        cmd.arg("libturbo.a");
+        cmd.arg("go-turbo");
     }
     assert!(
         cmd.stdout(std::process::Stdio::inherit())
             .status()
-            .expect("failed to build turbo.lib")
+            .expect("failed to build go binary")
             .success(),
-        "failed to build turbo static library"
+        "failed to build go binary"
     );
+
+    let go_binary_name = if target.os == build_target::Os::Windows {
+        "go-turbo.exe"
+    } else {
+        "go-turbo"
+    };
+
+    let go_binary_path = env::var("CARGO_WORKSPACE_DIR")
+        .map(PathBuf::from)
+        .unwrap()
+        .join("cli")
+        .join(go_binary_name);
+
+    let new_go_binary_path = env::var_os("CARGO_WORKSPACE_DIR")
+        .map(PathBuf::from)
+        .unwrap()
+        .join("target")
+        .join("debug")
+        .join(go_binary_name);
+
+    fs::rename(go_binary_path, new_go_binary_path).unwrap();
```

Comment:
Wouldn't copying be better?

Also, might want to add a `cargo:rerun-if-changed` to pick up changes